index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+</head>
+<body>
+    <section class="to-do-container">
+        <h1>To Do</h1>
+        <form action="#" id="to-do-form">
+            <input type="text" name="text" id="to-do-input" placeholder="Add to your list...">
+            <button id="submit-button"><i class="fas fa-check"></i></button>
```

Comment:
I suggest adding a type value to your button, as web hint flags me with this suggestion!